02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,86 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
   <link rel="stylesheet" href="css/style.css">
+  <title>Zac Collier Music</title>
+
+<!--  HEADER -->
```

Comment:
Careful! head and header aren't the same thing; the head is where the metadata for the page goes, such as the CSS. Everything you want to display to the user should go in the body--including the header!